### ISSUE TITLE
Time to update screenshots?

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium.github.io#43]
Quite a lot has happened with Stellarium the last decade. The screenshots shown at https://stellarium.org/screenshots.html are extremely outdated. A few ideas:

- AstroCalc, e.g. brightness graph
- DSO outlines
- Solar eclipse simulation
- HiPS/TOAST
- Script
- Plugin, e.g Archaeolines
- Some non-western sky culture

Just a few ideas.

**Comment by 10110111** [comment]
Indeed. Also, the format should be PNG, or at least a less compressed JPEG, because in the current form the screenshots have awful quality.

**Comment by gzotti** [comment]
Some images are certainly more than 10 year old, when bandwidth may have been more of a problem. Screenshot format "jpeg" is less compressed, or just make them larger. Certainly also worth depicting: 
- satellites/ISS
- scenery3d
- comets
- light pollution foreground
- exoplanet distribution
- closeup of occultation between Jupiter's moons. 

Should we keep the 0.10 images in a "museum"?

**Comment by 10110111** [comment]
> Should we keep the 0.10 images in a "museum"?

I don't find any value in them, especially given how crippled they are by the compression. If someone wants to see how Stellarium 0.10 looked, the download is still available in the "all releases" section.

**Comment by alex-w** [comment]
I fear we need new design for the website, not only updating screenshots and slideshow (hint: check img/screenshots directory). For a "museum" - please check https://dss.stellarium.org/stellarium/ - it's not complete yet, but it is almost a full mirror of packages.
